docs/section4/4-10.md

```diff
@@ -0,0 +1,38 @@
+# ASHVAC – Rule 4-10
+
+**Rule ID:** 4-10
+ 
+**Rule Description:** HVAC Fan Schedules. Schedules for HVAC fans that provide outdoor air for ventilation shall be cycled ON and OFF to meet heating and cooling loads during unoccupied hours.  
+**Rule Assertion:** P-RMR = expected value                                           
```

Comment:
For this and other rules, I would be more specific on which p_RMR element we are checking (operation_during_unoccupied). But perhaps what you have matches Xing's RDSs, I often don't pay attention to this part.

Reply:
I looked at Xing's and didn't find many where this was specific. Need to ask James about this.

Reply:
@claperle one general comment, for each RDS it would help to clarify in the rule description, whether the rules applies to P or B. We're using this to develop the Rule Test spreadsheet and can align the rule description in both.

Reply:
@supriyagoel Weird, that clip appears outdated, this what I had at my end "Schedules for HVAC fans that provide outdoor air for ventilation shall be cycled ON and OFF to meet heating and cooling loads during unoccupied hours excluding HVAC systems that meet Table G3.1-4 Schedules for the proposed building exceptions #s 2 and 3.  "